mmil.raw.html

```diff
@@ -9666,6 +9666,48 @@
   <td>adds conditions on ` M ` and ` N `</td>
 </tr>
 
+<tr>
+  <td>pfxnndmnd</td>
+  <td><i>none</i></td>
+  <td>the set.mm proof uses mpondm0 so it is possible set existence
+  conditions would be needed</td>
+</tr>
+
+<tr>
+  <td>pfx00</td>
+  <td>~ pfx00g</td>
+  <td>adds set existence condition</td>
+</tr>
+
+<tr>
+  <td>pfx0</td>
+  <td>~ pfx0g</td>
+  <td>adds ` L e. NN0 ` condition</td>
+</tr>
+
+<tr>
+  <td>pfxval0</td>
+  <td>~ pfxval</td>
+</tr>
+
+<tr>
+  <td>pfxcl</td>
+  <td>~ pfxclg</td>
+  <td>adds ` L e. NN0 ` condition</td>
```

Comment:
note to self: can trivially be generalized to ZZ, although it doesn't make sense